packages/belos/tpetra/src/solvers/Belos_Tpetra_Gmres.hpp

```diff
@@ -486,8 +488,9 @@ class Gmres : public Krylov<SC, MV, OP> {
 
     mag_type b_norm;  // initial residual norm
     mag_type b0_norm; // initial residual norm, not left preconditioned
-    mag_type r_norm;
+    mag_type r_norm, r_norm_imp;
```

Comment:
This is fine, but [prefer one declaration per line](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Res-name-one).